index.bs

```diff
@@ -3036,6 +3036,14 @@ optionally evidence of [=user consent=] to a specific transaction.
         The [=client=] and [=authenticator=] make a best-effort to create a credential of the most preferred type possible.
         If none of the listed types can be created, the {{CredentialsContainer/create()}} operation fails.
 
+        [=[RP]=]'s that wish to support a wide range of [=authenticators=] SHOULD define entries for the following discrete list of signature algorithms:
```

Comment:
In what way is this list "discrete"? I'm not sure what that word is supposed to mean here, I've only seen it in contexts like "discrete vs. continuous" or "discrete components vs. integrated circuits".

Reply:
I tried using "discrete" to communicate "a finite list" instead of "all possible algorithm IDs now and in the future."

Reply:
Ok. Either way, I guess we can close this as the term no longer appears in the latest version. :slightly_smiling_face: